### ISSUE TITLE
Are any changes still required for webp support

### ISSUE THREAD

**Issue by johnf** [facebook/react-native-website#3643]
## Description

The documentation at https://github.com/facebook/react-native-website/blob/main/docs/image.md#gif-and-webp-support-on-android
indicates that extra libraries are required in build.gradle to support webp on android.

## What is the problem?

Is this still accurate? I'm on latest RN 0.71.4 and webp worked for me without any changes.

## How can we address it?

I'm happy to create a PR to remove this section if it is no longer needed.

## Why is it important?

It indicates extra work is needed that may be unnecessary.

## Who needs this?

Android devs

## When should this happen (use version numbers if needed)?

Not super urgent

**Comment by cortinico** [comment]
> Is this still accurate? I'm on latest RN 0.71.4 and webp worked for me without any changes.

Yes it's still accurate. You need Fresco WebP module to be added as there are different types of WebP (e.g. transparent, animated, etc.) and you probably want to support a wider groupd of users.

You can read more about it here https://frescolib.org/docs/webp-support.html
